### PR TITLE
Added VirusTotal Integration, Unit Tests & dockerfile and K8 config

### PR DESCRIPTION
**Pull request by belwase**
- virustotal integration
- more unit tests 
- docker and k8 config

### CODE REVIEW

**Review by susilupwork — COMMENTED** [review]
- Virusshare.py, Virustotal : url, hash_url, BASEURL should be configurable, not hard coded in .py file. Write Metadata information should be configurable (Preferably via .env file or config files using configparser)
-  All File handling code should move to your file_processor.py or FileService.py.  Now IT is spread between this two classes, also inside the other class. 
- Naming should be as per Glasswell Python guideline.  Now in your code class is named in snake case as well as CameCase. e.g. file_processor.py or FileService.py 
- virustotal.py .file_scan() is doing file operation before checking whether the file is valid or not. 
- Also all file operation need prior validation. 
- All I/O , Network operation must have exception handling.
- In AppTestCase you are putting api_key value in code, should be taken env variable or config.
Can be done after Monday (Not priority for Thurseday pilot)
- Logging should be done with INFO, WARNING, ERROR etc. 
- All external communication or transaction must be tracked in database.

File: upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        expected = {"file_name": "test_file.png"}
+        mocked_scrape.return_value = "test_file.png"
+
+        response = self.client.post(endpoint, json={})
+        self.assertEqual(response.status_code, 400)
+        self.assertIn("api_key and hash required", response.get_json()["error"])
+
+        payload = {
+            "api_key": "12332839h98h2309823",
+            "hash": "0c02d22721b03f49339e7c89b6dd2cb8",
```
Comment (DinisCruz) [inline upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py:29]: Hi, don’t store passwords or API keys on Source code

File: upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        endpoint = "/scrape-vs"
+        response = self.client.post(endpoint)
+        self.assertEqual(response.status_code, 200)
+
+        # check file downloaded
+        self.assertTrue(os.path.exists(hash_file))
+        with open(hash_file, "r") as f:
+            self.assertIn("32893293", f.read())
+        os.system(f"rm {hash_file}")
```
Comment (DinisCruz) [inline upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py:79]: Great to see tests :)

**Review by DinisCruz — APPROVED** [review]
Great work, I’m going to merge this, but please implement the changes asked In the reviews of this PR

**Comment by DinisCruz** [comment]
Great reviews @susilupwork

File: upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py (hunk 1 above)
Comment (belwase) [reply upwork-devs/ashish-belwase/virusshare_poc/test/test_app.py:29]: @DinisCruz  These are dummy api_key for test

**Pull request merged**